src/api/controllers/admin.controller.js

```diff
@@ -0,0 +1,34 @@
+import * as _ from "lodash";
```

Comment:
This may import more modules than you need and increase your bundle size,

you can use this instead: `import { pick } from “lodash”;`